server/database/db_build.js

```diff
@@ -0,0 +1,11 @@
+const { readFileSync } = require("fs");
+const { join } = require("path");
+
+const connection = require("./connection");
+
+const sql = readFileSync(join(__dirname, "build.sql")).toString();
```

Comment:
Here you named the file build.sql 
but truly the file named db_build.sql, so the build throw an error